### ISSUE TITLE
Adding twitter bootstrap css reduces ribbon height and mis-aligns the text 

### ISSUE THREAD

**Issue by ddewaele** [simonwhitaker/github-fork-ribbon-css#50]
When bootstrap css is added to the page 

```
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```

The ribbons height is reduced and text is no longer aligned :

![](https://dl.dropboxusercontent.com/u/13246619/misc/ribbon.png)

**Comment by ddewaele** [comment]
PR https://github.com/simonwhitaker/github-fork-ribbon-css/pull/48 seems to solve it.

**Comment by simonwhitaker** [comment]
Fixed in #48

**Comment by darklynx** [comment]
@simonwhitaker do you plan to release the fixed version of CSS?
